content/exchange/artifacts/Exchange.Windows.System.PowerShell.DetectResponder

```diff
@@ -0,0 +1,34 @@
+name: Custom.Windows.System.PowerShell.DetectResponder
+author: "Dhruv Majumdar, Jamie Bhoohe"
+description: |
+  This artifact allows to detect responder in the environment
+type: CLIENT_EVENT
+required_permissions:
+  - EXECVE
+
+precondition:
+  SELECT OS From info() where OS = 'windows'
+
+parameters:
+  - name: Command
+    default: "dir C:/"
+  - name: PowerShellExe
+    default: "C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe"
```

Comment:
Is there a way we can do it without powershell? 

We try to not use powershell too much because it can add confusing messages to the powershell logs (especially with encoded commands).

for example
```sql
SELECT * FROM http_client(url="http://evildexter/")
WHERE Content =~ "no such host"
```

Reply:
We must use PowerShell for this code to work. Without it it will not work.